README.md

```diff
@@ -1,70 +1,45 @@
-# Getting Started with Create React App
+# Covid-Tracker
+A Covid Tracker Webapp with a mobile-first design that displays the present number of confirmed cases of Covid across selected countries, incorporating the Narrative API to a React-Redux framework.
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+<!-- ![screenshot](src/assets/screenshot.png) -->
 
-## Available Scripts
+## Live Demo
```

Comment:
Please deploy the project and add a link to the deployed project in the README.

Reply:
Deployed on Netlify, and added a link to the preview to the readme. The final address will be available when this pull request goes to the production branch.